features/step_definitions/hangout_steps.rb

```diff
@@ -182,4 +185,7 @@
   page.should have_field('hangout_url', with: @hangout_url)
 end
 
-
+Then(/^a separate event instance is not created$/) do
+  expect(EventInstance.where("created_at >= ?", Time.now.beginning_of_day).size)
```

Comment:
yeah, I guess you're getting beginning of day - which is almost constant there, unless the test took a day to run, and that's a silly corner case, so I guess we're okay, but in general you should avoid using Time like that in a test